bin/llvm-kompile

```diff
@@ -92,6 +90,11 @@ while [[ $# -gt 0 ]]; do
       kompile_clang_flags+=("-v")
       shift
       ;;
+    --profile)
```

Comment:
Shouldn't we print some error/warning when `--profile` is used without verbose?

Reply:
Sure, can do